src/ValueObject/ImageValueObject.php

```diff
@@ -124,7 +124,7 @@ public function getArray(): array {
    * @return $this
    */
   public static function fromImageItem(ImageItem $image_item): ValueObjectInterface {
-    $image_file = $image_item->get('entity')->getTarget();
+    $image_file = $image_item->get('entity')->getValue();
```

Comment:
Interesting. I thought we need `->getTarget()->getValue()`, but you found a shortcut. Good :)